src/trytond_factories/attachment.py

```diff
@@ -0,0 +1,37 @@
+
+__all__ = [
+    'DataAttachment',
+    'LinkAttachment',
+]
+
+import factory
+import factory_trytond
+
+
+class _Attachment(factory_trytond.TrytonFactory):
+    class Meta:
+        model = 'ir.attachment'
+
+    name = factory.Faker('word')
+    type = factory.Faker('word', ext_word_list=['data', 'link'])
```

Comment:
If downstream DataAttachment and LinkAttachment factories already have a fixed value for type, this can also be omitted. Otherwise I'd expect the generic typed Attachment factory to also have rich data, file_id and link declarations with https://factoryboy.readthedocs.io/en/stable/reference.html#traits